BillTechPaymentsUpdater.php

```diff
@@ -76,7 +76,7 @@ function updatePayments()
 		$last_sync = $this->getLastSync();
 
 		$client = BillTechApiClientFactory::getClient();
-		$path = "/pay/v1/payments/search" . "?fromDate=" . (ConfigHelper::checkConfig("billtech.debug") ? 0 : ($last_sync - 60));
+		$path = "/pay/v1/payments/search" . "?fromDate=" . (ConfigHelper::checkConfig("billtech.debug") ? 0 : ($last_sync - 3600));
```

Comment:
Po namyśle dałbym dobę. Godzinę to może trwać jakaś awaria.

Reply:
Ale jeśli awaria trwa dobę to i tak nie aktulizujemy last_sync. W last sync przetrzymywane jest informacja o ostatnim pozywytnym syncu. Czyli jak awaria trwa 12h to nie wykonamy żadnego pozytywnego synca to i tak złapiemy 12h+1h=13h. Boję się, że ta doba za bardzo może nas obciążać niepotrzebnie.